.github/workflows/cd-sql-engine-tests.yaml

```diff
@@ -91,6 +91,25 @@ jobs:
           additional-pytest-options: ${{ env.ADDITIONAL_PYTEST_OPTIONS }}
           make-target: "test-databricks"
 
+   trino-tests:
+    environment: DW_INTEGRATION_TESTS
+    name: Trino Tests
+    if: ${{ github.event.action != 'labeled' || github.event.label.name == 'Run Tests With Other SQL Engines' }}
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check-out the repo
+        uses: actions/checkout@v3
+
+      - name: Test w/ Python ${{ env.PYTHON_VERSION }}
+        uses: ./.github/actions/run-mf-tests
+        with:
+          python-version: ${{ env.PYTHON_VERSION }}
+          mf_sql_engine_url: ${{ secrets.MF_TRINO_URL }}
+          mf_sql_engine_password: ${{ secrets.MF_TRINO_PWD }}
```

Comment:
I don't think we need these, but I have to test. The test-trino make command works on its own as long as the service is up and running, since it's doing a local service pointer the same way our postgres tests run.